en/_index.md

```diff
@@ -2,85 +2,80 @@
 title: TiDB in Kubernetes Documentation
 summary: Learn about TiDB in Kubernetes documentation.
 aliases: ['/docs/tidb-in-kubernetes/dev/']
+hide_sidebar: true
+hide_commit: true
 ---
 
-# TiDB in Kubernetes Documentation
+<LearningPathContainer platform="tidb-operator" title="TiDB in Kubernetes Documentation" subTitle="Using TiDB Operator provided by PingCAP, you can run and maintain TiDB seamlessly in the Kubernetes clusters deployed on a public cloud or on premise.">
```

Comment:
"[on-premises](https://andrewschoen.medium.com/software-people-of-the-world-stop-saying-on-premise-you-sound-dumb-56692dae871d#:~:text=%E2%80%9CON%2DPREMISE%E2%80%9D-,TLDR%3A%20It's%20%E2%80%9Con%2Dpremises%E2%80%9D%20or%20%E2%80%9Con%2D,say%20%E2%80%9Con%2Dpremise.%E2%80%9D)" is more accurate.
```suggestion
<LearningPathContainer platform="tidb-operator" title="TiDB in Kubernetes Documentation" subTitle="Using TiDB Operator provided by PingCAP, you can run and maintain TiDB seamlessly in the Kubernetes clusters deployed on a public cloud or on premises.">
```